openwisp_controller/config/static/config/css/lib/jsonschema-ui.css

```diff
@@ -294,9 +294,6 @@ div.jsoneditor > div > h3.controls{
   position: static;
   margin-top: 15px;
 }
-.jsoneditor-wrapper div.jsoneditor .grid-row > .grid-column > label {
-  display: none;
-}
```

Comment:
@nemesisdesign do you remember why we added this? I checked the WireGuard configuration in the Configuration Editor and I don't see any negative effects from removing this CSS.

Reply:
I don't remember :disappointed: 
I am double checking now.

Reply:
This needs more work. I will handle it since I don't know how to explain it. Feel free to ping me from time to time.